cmd/platform.go

```diff
@@ -28,6 +28,8 @@ func configurePlatform(cmd *cobra.Command) {
 	flags.StringSliceP("group", "G", nil, "The name of a GitLab organization. All repositories in that group will be used.")
 	flags.StringSliceP("user", "U", nil, "The name of a user. All repositories owned by that user will be used.")
 	flags.StringSliceP("repo", "R", nil, "The name, including owner of a GitHub repository in the format \"ownerName/repoName\".")
+	flags.StringP("repo-search", "", "", "Use a repository search to find repositories to target.")
+	flags.StringP("repo-exclude", "", "", "Exclude certain repositories with a regex string")
```

Comment:
Would you like this to `filter` (_include_ only those that match this regex) or `exclude` when this matches.

`exclude` may be useful when used with the `repo-search`, but `filter` may be more useful if the `org` or `user` targeting is used.

Food for thought.